test-jest/polar/Radar.spec.tsx

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+import { uniqueId } from 'lodash';
+import { render, screen } from '@testing-library/react';
+
+import { Surface, Radar } from '../../src';
+
+type point = { x: number; y: number };
```

Comment:
Types should always be PascalCase
```suggestion
type Point = { x: number; y: number };
```